org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -89,6 +90,88 @@ class Translator {
         return this._makeRequest(requestPath, 'PUT', putPayloadString);
     }
 
+    /**
+     * Subscribes to realtime notifications for this provider.
+     * 
+     * callback is the function that will be called with the JSON ocf representation of the changed device
+     */
+    subscribe(callback) {
```

Comment:
in the hub raml file, add another method like so:

/SubscribeHubResURI, make it a post, the request body should have a blob (it'll be a callback but for raml make it a blob), and the response should be the same as the get. the response ties in with the what the user will see on the notification.  it's a convention we are enabling through our javascript implementation

---
In reply to: [79207243](https://github.com/openT2T/translators/pull/99#discussion_r79207243) [](ancestors = 79207243)